src/routes/safe/components/Apps/components/AppsList.test.tsx

```diff
@@ -1,28 +1,32 @@
-import AppsList from './AppsList'
-import { render, screen, fireEvent } from 'src/utils/test-utils'
+import AppsList, { PINNED_APPS_LIST_TEST_ID, ALL_APPS_LIST_TEST_ID } from './AppsList'
+import { usePinnedSafeApps } from '../hooks/appList/usePinnedSafeApps'
+import { render, screen, fireEvent, within, act, waitFor } from 'src/utils/test-utils'
+
+jest.mock('src/routes/safe/components/Apps/hooks/appList/usePinnedSafeApps')
```

Comment:
What is your opinion about to mock the Apps endpoints & localStorage instead of the hooks directly?

we can mock the axios.get call in the `fetchSafeAppsList` in `/src/logic/configService/index.ts` file and the localStorage keys like `_immortal|v2_RINKEBY__PINNED_SAFE_APP_IDS`.
We can also use generic test apps like appTest1, appTest2... instead of real data like Drain safe or Compound app

The main benefit here is that we can test also the hooks! (but we can create a different ticket to address this if you want)